src/command/schema/push.rs

```diff
@@ -0,0 +1,70 @@
+use crate::client::get_rover_client;
+use anyhow::Result;
+use rover_client::query::schema::push;
+use std::path::{Path, PathBuf};
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+pub struct Push {
+    /// where to find the schema. .graphql, .json or uri
+    #[structopt(name = "SCHEMA_PATH", parse(from_os_str))]
+    schema_path: PathBuf,
+    /// The variant of the request graph from Apollo Studio
+    #[structopt(long, default_value = "current")]
+    variant: String,
+    #[structopt(long)]
+    graph: String,
```

Comment:
```suggestion
    #[structopt(long)]
    graph_name: String,
```

this should be `graph_name` to match what is in `schema get`.

i'm also guessing that it should be a required positional argument since you kinda need to know what graph you're pushing a schema to (unless not specifying a graph is a valid operation, in which case this is fine)